packages/react-components/react-input/README.md

```diff
@@ -3,3 +3,35 @@
 **React Input components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
 
 These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+Inputs give people a way to enter and edit text.
+
+### Usage
+
+Import Input:
+
+```js
+import { Input } from '@fluentui/react-input';
```

Comment:
Should this reference react-components instead?
```suggestion
import { Input } from '@fluentui/react-components';
```

Reply:
I'm going to make that change once I move the component to RC and it's available at that path.